modules/ROOT/pages/external-oauth-2.0-token-validation-policy.adoc

```diff
@@ -1,43 +1,173 @@
+|===
+| Element | Description | Example
+| Scopes | Displays a space-separated list of supported scopes. | `READ`, `WRITE`, `READ and WRITE`
+| Scope Validation Criteria | Determines if the token must contain all defined scopes or just one.
+Values: `Contains all scopes` and `Contains any scope` | 
+If the ‘Contains all scopes’ value is selected, the tokens with the scopes `READ` and `WRITE` are accepted. If the ‘Contains any scopes’ value is selected, the tokens with the scope `READ`, `WRITE`, and `READ AND WRITE` are accepted.
+| Access Token validation endpoint url | The url of the Access Token validation endpoint of the External OAuth2 Provider. |The link to a mule provider hosted by the client: http://client-hostetd-mule-provider-url.cloudhub.io/validate
+| Validate TLS Certificate | Enables third-party authentication server TLS validation. | If selected, the communication with the token validation endpoint uses the TLS protocol.
+| Expose Headers | In a proxy scenario, specifies whether headers must be exposed in the request to the backend. User properties returned by the federation server are prefixed with 'X-AGW-' and sent as headers to the backend. | See the <<leveraging-TVE-information,Leveraging Token Validation Endpoint Information for Mule Applications>> for more information about this element.
+| Skip Client Id Validation | Skips the client application's API contract validation. | If selected, the Step 4 of the diagram in the <<how-this-policy-works,How This Policy Works>> section is skipped.
+|===
+
+== FAQs
+
+*Can I generate OAuth2 tokens with the policy?*
+
+No. The policy only validates the tokens.
+
+*Can I use other Oauth providers with the policy?*
+No. This policy is designed to exclusively work with thexref:mule-oauth-provider-landing-page.adoc[Mule Oauth provider]. Using the policy with any other OAuth 2.0 provider (for example, Facebook, Google, or Azure) is not supported.
+
+*Mule OAuth provider validation endpoint is rejecting the requests sent by the policy.*
+
+If the validation endpoint is rejecting requests, verify whether the URL configured in the policy is reachable from the machine where the policy is running and that no authentication is required to successfully complete the request. If your use case requires the validation endpoint to be protected, you can switch to the xref:policy-openid-connect.adoc[Open ID connect policy].
+
+
+*Where are the OAuth tokens cached?*
+
+OAuth2 tokens are cached only in memory and are never written to disk.
+
+
+*Does the policy communicate with the Anypoint Platform on a per-request basis?*
+
+No, the policy uses client applications previously retrieved by the Mule runtime engine.
+This enables the policy to continue working even if the connection with the Management plane is lost. If you want the client application information that is written on disk to be encrypted, xref:api-gateway-encryption-mule4.adoc[configure gateway encryption in your runtime].
```

Comment:
We do not capitalize the names of planes, per our source for product naming:
```suggestion
This enables the policy to continue working even if the connection with the management plane is lost. If you want the client application information that is written on disk to be encrypted, xref:api-gateway-encryption-mule4.adoc[configure gateway encryption in your runtime].
```

Reply:
Done! Thanks!